frontend/webpack.config.js

```diff
@@ -45,7 +45,7 @@ module.exports = {
     rules: [{
       test: /\.jsx?$/,
       enforce: 'pre',
-      exclude: [/node_modules/],
+      exclude: [/node_modules/, /frontend_lib/],
```

Comment:
It is not the case when yarn is used in PNP mode: frontend_lib is not in a node_modules folder because there aren't node_modules folders anymore in this mode. This is why I put the additional regular expression. The regular expression `/node_modules/` is going to be useless. I suggest keeping both regular expressions for now so Tracim is compatible with both PNP mode and regular `node_modules` folders and we just have to figure out Cypress to enable PNP.

Reply:
Ok, as long as frontend_lib is not listed as local dependency in package.json, it should be fine.

I just don't understand how yarn handles it, we'll can talk about it later.

Afaik, we currently don't use pnp mode because of cypress. So we need to keep /node_modules/. When it'll not be required, we will have to remove it though